pvanalytics/system.py

```diff
@@ -547,6 +574,10 @@ def infer_orientation_fit_pvwatts(power_ac, ghi, dhi, dni,
         ``pvlib.temperature.TEMPERATURE_MODEL_PARAMETERS['sapm'][
         'open_rack_glass_glass']`` is used. See
         :py:func:`pvlib.temperature.sapm_cell` for more information.
+    azimuth_min: Float, default 0
+        Minimum possible azimuth (bounds) for the least squares search problem.
+    azimuth_max: Float, default 360
+        Maximum possible azimuth (bounds) for the least squares search problem.
```

Comment:
If we're exposing these as optional parameters, should the other bounds (tilt's 0 to 90, e.g.) be exposed as well?

Also, `[degrees]` at the ends of the descriptions